src/CompletionCommand.php

```diff
@@ -15,11 +15,15 @@ class CompletionCommand extends SymfonyCommand
      */
     protected $handler;
 
+    /**
+     *  Bash completion command description.
+     */
+    protected static $defaultDescription = 'BASH completion hook.';
```

Comment:
where are we using this variable?

Reply:
This was deprecated in 6.1 and removed in 7.0. Please use `setDescription` as before

Reply:
Done

Reply:
Done.